reports/tests.py

```diff
@@ -90,6 +96,13 @@ def has_listeners(class_name):
         post_save.connect(receiver=model_saved,
                           dispatch_uid='instance-saved-hook')
 
+        try:
+            with mock.patch('random.choice', mockobj.choice):
+                filename = generate_random_filename()
+                os.remove(filename)
```

Comment:
This is to actually delete the file, I mock choice to get the same filename as the tests. It only generates a name, not the actual file.